aten/src/ATen/native/cuda/ForeachBinaryOpScalarList.cu

```diff
@@ -67,8 +68,53 @@ std::vector<Tensor> foreach_tensor_##NAME##_scalarlist_kernel_cuda(TensorList te
 }
 
 FOREACH_BINARY_OP_SCALARLIST(add, std::plus);
-FOREACH_BINARY_OP_SCALARLIST(sub, std::minus);
 FOREACH_BINARY_OP_SCALARLIST(mul, std::multiplies);
-FOREACH_BINARY_OP_SCALARLIST(div, std::divides);
+
+// In the case of division, integer inputs will result in float. 
+// Currently multi tensor apply can only return result of the same type as input.
```

Comment:
nit: It would be good to mention "This does not use FOREACH_BINARY_OP_SCALARLIST because <everything else in the comment>". It's easy to make the inference but just to clarify

Reply:
Done